Projects/UOContent/Items/Armor/Leather/GargishLeatherArms.cs

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Remove extra line on all files that have it. (I also wish we had a good linter to do this automatically)